keyvault/client/keys/changelog.md

```diff
@@ -0,0 +1,5 @@
+# Release History
```

Comment:
Consider using uppercase filename `CHANGELOG` to be consistent with other files like README, CONTRIBUTING, LICENSE etc.

Reply:
renamed.
But some SDKs like Batch and Cosmos were using lowercase.
Lets go uppercase as that's the standard.